PocketLobby/constituents
# Show matches

[Trello Card](TRELLO_TICKET_URL)

To test this:

1) make sure you have legislators populated: `docker-compose exec constituent_rails rake legislators:populate`
2) add some bill summaries with slugs of bills that have passed at least one chamber from the 116th congress `https://www.congress.gov/search?q=%7B%22source%22%3A%22legislation%22%2C%22congress%22%3A%22116%22%2C%22bill-status%22%3A%22passed-one%22%7D` (should be prefiltered)
3) go to campaigns you added bills to and vote
4) go to `/matches`. You should see a table with your rep and two senators, and how you matched up to them!

NOTES:

- It's ugly af
- we need some basic UI friendliness like a title and navigation to get there
- looking forward, maybe table becomes interactive / we can add some general payment stuff on this page
- I copy pasted the query from postico so maybe the spacing is sorta fucked


## Prerequisite PRs



## Dependent PRs



## New ENV Vars



## Notes for Reviewer
